docs/gaia/launch/blog-1-sp.md

```diff
@@ -0,0 +1,181 @@
+desconocido en el contexto de cadenas de bloques o cryptocurrency. Un atacante puede intentar atraerlo a 
+un [sitio comprometido](https://blog.malwarebytes.com/cybercrime/2013/02/tools-of-the-trade-exploit-kits/) diseñado para robar información confidencial de su equipo. Si eres usuario de Gmail, prueba tu 
+capacidad de recuperación frente a las últimas tácticas de phishing basadas en correo electrónico[aquí](https://phishingquiz.withgoogle.com/).
+
+- Haga su debida diligencia antes de comprar Atoms. Los Atoms no serán transferibles en el momento del lanzamiento, por lo 
+que no pueden ser comprados o vendidos hasta que un hard-fork lo permita. Cuando sean transferibles, asegúrese de que ha 
+investigado al vendedor o intercambiador para confirmar que los Atoms provienen de una fuente confiable.
+
+- Ni el equipo de Tendermint ni la Fundación Interchain venderán Atoms, así que si ves mensajes en los medios sociales o 
+correos electrónicos que anuncian una venta simbólica de nosotros, no son reales y deben ser evitados.  Habilite la 
+autenticación de 2 factores y tenga en cuenta los métodos de recuperación utilizados para recuperar el acceso a sus cuentas 
+más importantes. Las cuentas sin protección como el correo electrónico, los medios sociales, tu cuenta de GitHub, el Foro 
+Cosmos y cualquier otra cosa en el medio podrían dar a un atacante oportunidades de ganar terreno en tu vida en línea. Si 
+aún no lo ha hecho, empiece a utilizar inmediatamente una aplicación de autenticación o una hardware key dondequiera que 
+manejes tus tokens.  Esta es una manera simple, efectiva y comprobada para reducir el riesgo de robo de cuenta.
+
+- Sea escéptico respecto a los consejos técnicos, especialmente si provienen de personas que no conoce en los foros y en los 
+canales de chat grupal. Familiarícese con los comandos importantes, especialmente aquellos que le ayudarán a llevar a cabo 
+acciones de alto riesgo, y consulte nuestra documentación oficial para asegurarse de que no se le está engañando para que 
+haga algo que pueda perjudicarle a usted o a su validador. Y recuerda que el foro del Cosmos, los canales Riot y Telegram no 
+son fuentes de información oficial o de noticias sobre Cosmos. 
+
+- Verifique las transacciones antes de pulsar enviar. Sí, esas secuencias de direcciones son largas, pero comparándolas 
+visualmente en bloques de 4 carácteres a la vez puede ser la diferencia entre enviarlas al lugar correcto o enviarlas al 
+olvido.
+
+*Si aparece un acuerdo que [suena demasiado bueno para ser cierto][good], o aparece un mensaje pidiendo información que 
+nunca, nunca, nunca debe ser compartida con otra persona, siempre puede esforzarse por verificarla antes de comprometerse 
+con ella navegando a un sitio web o canal de comunicación oficial de Cosmos por su cuenta. Nadie de Cosmos, el equipo de 
+Tendermint o la Fundación Interchain enviarán alguna vez un email pidiéndole sus 12 palabras para compartirlas con nosotros, 
```

Comment:
```suggestion
Tendermint o la Fundación Interchain enviarán alguna vez un email pidiéndote sus 12 palabras secretas para compartirlas con nosotros, 
```

Reply:
Same as above, it's more correct to talk to them as _"usted"_  instead of _"tú"_.
But if you think it's more correct like you say I can change it.